tests/conftest.py

```diff
@@ -19,7 +19,8 @@ def extract_strings(vw):
     decoding_functions_candidates = identify_decoding_functions(vw)
     decoded_strings = floss_main.decode_strings(vw, function_index, decoding_functions_candidates)
     selected_functions = floss_main.select_functions(vw, None)
-    decoded_stackstrings = stackstrings.extract_stackstrings(vw, selected_functions)
+    bb_ends = stackstrings.get_basic_block_end_index(vw)
+    decoded_stackstrings = stackstrings.extract_stackstrings(vw, selected_functions, bb_ends)
```

Comment:
i think we can move the generation of the ends index into the `extract_stackstrings` function.

we don't reuse the index, so there's no benefit of keeping around.
also, as a user of the library, its just another thing you have to remember to do (construct the index and provide it as a parameter).